src/components/NewDraftWizard/WizardComponents/WizardFooter.js

```diff
@@ -5,18 +5,33 @@ import { useDispatch, useSelector } from "react-redux"
 import Link from "@material-ui/core/Link"
 import { Link as RouterLink } from "react-router-dom"
 import Button from "@material-ui/core/Button"
-import { decrement, increment, reset } from "../../../features/wizardStepSlice"
+import { reset } from "../../../features/wizardStepSlice"
```

Comment:
you can also use absolute imports: `mport { reset } from "features/wizardStepSlice"`